Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,31 @@
 import React from 'react';
 import './Button.css';
+
+// const NumButton = [
+//     <button className="number-button" >1</button>
+//     <button className="number-button" >2</button>
+//     <button className="number-button" >3</button>
+//     <button className="number-button" >4</button>
+//     <button className="number-button" >5</button>
+//     <button className="number-button" >6</button>
+//     <button className="number-button" >7</button>
+//     <button className="number-button" >8</button>
+//     <button className="number-button" >9</button>
+// ]
+
+const NumButton = <button>{}</button>
+const OpButton = <button>{}</button>
+
+const NumberButton = props => {
+    return (
+        <div>
+            <button className="number-button">{NumButton.props.text}</button>
+            <button className="number-button">{NumButton.props.text}</button>
+            <button className="number-button">{NumButton.props.text}</button>
+            <button className="operator-button">{OpButton.props.text}</button>
```

Comment:
You can take out this entire block of code and replace it with this:
```
<button className={props.class}>{props.text}</button>
```

Also, remember to take out the surrounding `<div></div>` tags too. The `<div>` will create a new row, but we do not want that (remember that the `NumberButton` component should be for an individual button, not for rows of buttons).

Therefore, all we have to return is a single `<button>` element using the props needed for its styling (`className={props.class}`) and text (`{props.text}`).